x-pack/plugins/fleet/server/services/fleet_server/elasticsearch/fleet_actions.json

```diff
@@ -24,6 +24,32 @@
       },
       "type": {
         "type": "keyword"
+      },
+      "user" : {
+        "properties": {
+          "id" : {
+            "type" : "keyword"
+          },
+          "email" : {
+            "type" : "keyword"
+          },
+          "full_name" : {
```

Comment:
Do we really need the text parts of ECS? Will we run text queries on these?

Reply:
This conforms to ECS definition of the fields.
Can add user.id only for sure.
Thought the extra user fields make the actions easier to render on UI without cross referencing between the indices.
Denormalization of the data seems pretty usual with nosql dbs